utils/create_sample_app.sh

```diff
@@ -0,0 +1,4 @@
+export PROJECT_HOME=$(pwd)
+cp -r ${PROJECT_HOME}/utils/sampleApp ${PROJECT_HOME}/src/SnippetApps
+
+## Add code here to update the app registry json
```

Comment:
```suggestion
## Add code here to update the app registry json

```

Yo! Let VSCode automatically do this for you https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files